gc/base/Collector.cpp

```diff
@@ -259,9 +259,14 @@ MM_Collector::preCollect(MM_EnvironmentBase* env, MM_MemorySubSpace* subSpace, M
 	}
 
 	/* If this is a global collection then set the globalgc flag.  This will allow us to
-	 * trigger excessiveGC checks from a local collection that has recursed into a global gc .
+	 * trigger excessiveGC checks from a local collection that has recursed into a global GC.
+	 * We are really interested in GCs that collect the whole heap since it's the point we can
```

Comment:
remove use of first person similar to what was done to the last sentence.
```suggestion
	 * Only include GCs that collect the whole heap to correctly evaluate the reclaimed
	 * memory criteria.
```